cdk/assets/scheduled_task_exec/index.js

```diff
@@ -1,38 +1,41 @@
-const aws = require('aws-sdk')
+const aws = require('aws-sdk');
 
-const ECS_CLUSTER = process.env.ECS_CLUSTER;
-const ECS_SERVICE = process.env.ECS_SERVICE;
-const ECS_TASK_DEFINITION = process.env.ECS_TASK_DEFINITION;
+const { ECS_CLUSTER } = process.env;
+const { ECS_SERVICE } = process.env;
+const { ECS_TASK_DEFINITION } = process.env;
 
 const ecs = new aws.ECS();
 
-exports.handler = async (event, context) => {
+exports.handler = async (event) => {
```

Comment:
I'm not sure what the rules are for this project, but some JSDoc on functions might be nice